src/app/CMakeLists.txt

```diff
@@ -161,10 +161,12 @@ endif (GUI AND WIN32)
 target_link_libraries(${QBT_TARGET_NAME} ${QBT_TARGET_LIBRARIES} QtSingleApplication::QtSingleApplication)
 
 if (APPLE)
-    set(qbt_BUNDLE_NAME "${CMAKE_PROJECT_NAME}")
+    set(qbt_BUNDLE_NAME "${QBT_TARGET_NAME}")
+    execute_process(COMMAND bash -c "sed -e \"s,@EXECUTABLE@,${qbt_BUNDLE_NAME},g\" ${qBittorrent_SOURCE_DIR}/dist/mac/Info.plist > ${qBittorrent_BINARY_DIR}/Info.plist.in")
```

Comment:
a `configure_file` call would be a cleaner option:
```cmake
set(EXECUTABLE  ${qbt_BUNDLE_NAME})
configure_file(${qBittorrent_SOURCE_DIR}/dist/mac/Info.plist ${qBittorrent_BINARY_DIR}/Info.plist.in @ONLY)
```
Also why does the final file has ".in" extension? Shouldn't it be "plist.in → .plist" transformation?